src/main/scala/com/amazon/deequ/analyzers/catalyst/StatefulKLLSketch.scala

```diff
@@ -0,0 +1,91 @@
+package org.apache.spark.sql
+
+import java.nio.ByteBuffer
+
+import com.amazon.deequ.analyzers.QuantileNonSample
+import com.amazon.deequ.analyzers.catalyst.KLLSketchSerializer
+import com.google.common.primitives.Doubles
+import org.apache.spark.sql.expressions.{MutableAggregationBuffer, UserDefinedAggregateFunction}
+import org.apache.spark.sql.types._
+
+
+private [sql] class StatefulKLLSketch(sketchSize: Int,
```

Comment:
Please format according to our [coding conventions](https://github.com/databricks/scala-style-guide)

Reply:
Done.